spec/features/saml/saml_spec.rb

```diff
@@ -268,5 +268,33 @@
         Timecop.return
       end
     end
+
+    context 'without SLO implemented at SP' do
+      let(:logout_user) { create(:user, :signed_up) }
+
+      before do
+        visit sp1_authnrequest
+        authenticate_user(logout_user)
+      end
+
+      it 'completes logout at IdP' do
+        allow_any_instance_of(ServiceProvider).to receive(:metadata).and_return(
+          {
+             service_provider: "https://rp1.serviceprovider.com/auth/saml/metadata",
+             authn_context: "http://idmanagement.gov/ns/assurance/loa/1",
+             last_authenticated_at: "2016-05-24 16:29:25",
+             user_id: 1,
+             created_at: "2016-05-24 16:29:25",
+             updated_at: "2016-05-24 16:29:25",
+             session_index: 1,
+             session_uuid: "_7672a58d-eee4-42fd-8e15-1c04d1ef3340",
+             quiz_started: false
+          })
+
+        visit destroy_user_session_url
+
+        expect(page.current_path).to eq('/')
```

Comment:
Good point, this could be improved. We have traditionally made a new request to ensure that the app is responding with intended behavior. I think I will move this to a controller spec.